README.md

```diff
@@ -4,6 +4,9 @@
 [![CodeQL](https://github.com/cisagov/skeleton-python-library/workflows/CodeQL/badge.svg)](https://github.com/cisagov/skeleton-python-library/actions/workflows/codeql-analysis.yml)
 [![Coverage Status](https://coveralls.io/repos/github/cisagov/skeleton-python-library/badge.svg?branch=develop)](https://coveralls.io/github/cisagov/skeleton-python-library?branch=develop)
 
+[![Code Style](https://img.shields.io/badge/Code%20Style-Black-black
```

Comment:
Is there a reason we have this badge on its own line?

Reply:
Please see commit 45be309.